src/app/menu/menu-admin/menu-admin.component.ts

```diff
@@ -28,7 +28,22 @@ export class MenuAdminComponent implements OnInit {
   private assignNewUsername() {
     this.authService.onIsAuth(
       () => {
-        this.isAdmin = true;
+        this.authService.getUserProfile().then((name: string): void => {
+          this.isAdmin = false;
```

Comment:
Is one user allowed to have more than one profile? Not for now - so it's better to create an enum and use one value only!